pkg/plugins/app_plugin.go

```diff
@@ -84,11 +84,6 @@ func (app *AppPlugin) InitApp(panels map[string]*PanelPlugin, dataSources map[st
 	cfg *setting.Cfg) []*PluginStaticRoute {
 	staticRoutes := app.InitFrontendPlugin(cfg)
 
-	// force enable bundled catalog app
-	if app.Id == "grafana-plugin-catalog-app" && cfg.CatalogAppEnabled {
-		app.AutoEnabled = true
```

Comment:
it is now always auto enabled... but the behavior is gated by `cfg.CatalogAppEnabled`

Controlling autoEnabled does not work once the app is in the database